src/utils/types.ts

```diff
@@ -17,7 +17,13 @@ export type FormErrorState = {
   fieldErrors: Record<string, string[]>;
 };
 
+export type Session = {
```

Comment:
If `sessionId` becomes part of this object then we can remove some function parameters where we are passing both the `sessionId` and the `Session`.